docker-compose.yml

```diff
@@ -82,6 +73,7 @@ services:
       - cache
     env_file:
       - api/env.docker
+      - api/.env
```

Comment:
Great idea! I like this for longer-lasting changes and I was always frustrated by having to jettison changes to `api/env.docker`.

Reply:
Also, risky! Too easy to accidentally commit secrets if I put them in that file :grimacing: